inst/include/Rcpp/sugar/functions/strings/trimws.h

```diff
@@ -133,7 +133,8 @@ inline Vector<STRSXP> trimws(const Vector<STRSXP>& x, const char* which = "both"
 }
 
 inline Matrix<STRSXP> trimws(const Matrix<STRSXP>& x, const char* which = "both") {
-    R_xlen_t i = 0, nr = x.nrow(), nc = x.ncol(), sz = x.size();
+    R_xlen_t i = 0, sz = x.size();
+    int nr = x.nrow(), nc = x.ncol();
```

Comment:
Per https://stat.ethz.ch/R-manual/R-devel/library/base/html/LongVectors.html 

> Arrays (including matrices) can be based on long vectors provided each of their dimensions is at most 2^31 - 1: 

Thus, each value is bigger than `UINT_MAX` 65,535 // 2^16 − 1. The `ULONG_MAX` 4,294,967,295 // 2^32 − 1
 has enough.